apps/site/pages/docs/customizing/mixins-breakpoints.mdx

```diff
@@ -0,0 +1,246 @@
+  <Tab>
+    ```bash copy
+    touch src/styles/custom-mixins.scss
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy
+    touch src\styles\custom-mixins.scss
+    ```
+  </Tab>
+</Tabs>
+
+3. Copy and paste the following code snippet into the file:
+
+```scss {8} filename="src/styles/custom-mixins.scss" copy
+@import "@faststore/ui/utilities"
+
+$breakpoints: (
+  "phone":         320px,
+  "phonemid":      375px,
+  "tablet":        768px,
+  "notebook":      1280px,
+  "desktop":       1500px,
+);
+```
+
+<Callout type="warning" emoji="⚠️">
+  You need to declare all the variants (phone, phonemid, tablet, notebook and desktop).
+</Callout>
+
+4. Run `yarn dev` in the terminal to start the development server.
+
+---
+
+## Mixins
+
+We use the [Sass `@mixin`](https://sass-lang.com/documentation/at-rules/mixin/) to allow you to define styles that can be re-used throughout your stylesheet. We divide them into these categories:
+
+
+<Tabs items={['Global mixins', 'Layout & Spacing Mixins', 'Focus Ring Mixins']}>
+  <Tab>
+    ```scss copy
+    $base: 16px !default;
+
+    @function rem($size) {
+      $rem: math.div($size, $base);
+
+      @return #{$rem}rem;
+    }
+
+    @mixin truncate-title($max-lines: var(--fs-text-max-lines)) {
+      display: -webkit-box;
+      overflow: hidden;
+      -webkit-box-orient: vertical;
+      -moz-box-orient: vertical;
+      -webkit-line-clamp: $max-lines;
+      line-clamp: $max-lines;
+      text-overflow: -o-ellipsis-lastline;
+      text-overflow: ellipsis;
+      white-space: normal;
+    }
+    ```
+  </Tab>
+  <Tab>
+    ```scss copy
+    @mixin layout-content {
+      box-sizing: border-box;
+      width: calc(100% - var(--fs-grid-padding) - var(--fs-grid-padding));
+      max-width: var(--fs-grid-max-width);
+      margin-right: auto;
+      margin-left: auto;
+    }
+
+    @mixin layout-content-full {
+      width: 100%;
+      max-width: 100%;
+    }
+
+    @mixin layout-colored($bkg-color: var(--fs-color-neutral-bkg), $padding-top: var(--fs-spacing-5), $padding-bottom: var(--fs-spacing-7)) {
+      padding-top: $padding-top;
+      padding-bottom: $padding-bottom;
+      background-color: $bkg-color;
+    }
+
+    @mixin section-divisor($position: "top", $padding: #{var(--fs-grid-padding)}) {
+      @if $position == "top" {
+        padding-top: $padding;
+        border-top: var(--fs-border-width) solid var(--fs-border-color-light);
+
+        @include media(">=notebook") { padding-top: $padding; }
+      }
+
+      @if $position == "bottom" {
+        padding-bottom: $padding;
+        border-bottom: var(--fs-border-width) solid var(--fs-border-color-light);
+
+        @include media(">=notebook") { padding-bottom: $padding; }
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+    ```scss copy
+    @mixin input-focus-ring($outline: #{var(--fs-color-focus-ring)}, $border: #{var(--fs-border-color-active)}) {
+      @media not all and (min-resolution: .001dpcm) { // Target only Safari browsers
```

Comment:
Actually, I think this is outdated: `Target only safari browser code`. We need to verify.